src/skin/popup.html

```diff
@@ -128,11 +128,24 @@ <h2 id="title-name"><span class="i18n_name"></span></h2>
       <span class="i18n_popup_instructions_no_trackers" data-i18n_contents_placeholders="<a target='_blank' title='i18n_what_is_a_tracker' class='tooltip' href='https://privacybadger.org/#What-is-a-third-party-tracker'>"></span>
       <span id="no-third-parties" class="i18n_popup_blocked" style="display:none"></span>
     </span>
+    <div id="toggle-blocked-resources-container">
+      <span id="expand-blocked-resources" class="ui-icon ui-icon-caret-d"></span>
+      <span id="collapse-blocked-resources" class="ui-icon ui-icon-caret-u"></span>
+    </div>
   </p>
   <div class="spacer"></div>
   <div id="blockedResources"></div>
 </div>
 
+<div id="firstparty-protections-container">
+  <div id="instructions-firstparty-protections" class="i18n_popup_info_firstparty_protections popup_info"></div>
```

Comment:
I think we should center the section headers ("X potential trackers blocked", "Link tracking protection is enforced").

Reply:
Could we make it easier to expand/collapse? The carets are pretty small click/tap targets. Maybe make it so that we could click/tap the entire header?